getting-started-with-directus-and-flask/index.md

```diff
@@ -0,0 +1,396 @@
+
+
+def get_global_data():
+    response = requests.get(f"{DIRECTUS_BASE_URL}/items/global")
+    return response.json().get("data")
+```
+
+By now this is all you need, but in the following sections, you will also create a new function to fetch data from other collections.
+
+## Rendering The Home Page
+
+To render the site home page, create a new route that uses the directus module to get the global data and use it on a page template.
+
+### Creating Page Templates
+
+First of all, you have to create a base template to be used by all your pages. Create a *templates* directory and a file called *base.html* in it. Now place the following content on this file:
+
+```jinja
+<!DOCTYPE html>
+<html lang="en" data-theme="dark">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link
+			rel="stylesheet"
+			href="https://cdn.jsdelivr.net/npm/@picocss/pico@2/css/pico.min.css"
+		/>
+		{% block head %}{% endblock %}
+
+		<title>{% block title %}Directus blog post with Flask{% endblock %}</title>
+	</head>
+	<body>
+		<header class="container">
+			<nav>
+				<ul>
+					<li>
+						<a href="/"><strong>Home</strong></a>
+					</li>
+				</ul>
+				<ul>
+					<li><a href="/blog">Blog</a></li>
+					<li><a href="/about">About</a></li>
+					<li><a href="/conduct">Code of Conduct</a></li>
+					<li><a href="/privacy">Privacy Policy</a></li>
+				</ul>
+			</nav>
+		</header>
+
+		<main class="container">{% block content %}{% endblock %}</main>
+	</body>
+</html>
+```
+
+:::info
+
+Note there is a stylesheet in this code that imports [Pico](https://picocss.com/), a minimal class-less CSS library used to add some basic styles on the website. It was only used on this article to make things look better, but you can feel free to use whichever library you prefer and even use the default browser styles.
+
+:::
+
+Now create a *templates/home.html* file that will extend the base template and display some data on it. This file will have the following content:
+
+```jinja
+{% extends "base.html" %}
+
+{% block content %}
+<article>
+	<h1>{% block title %}{{ title }}{% endblock %}</h1>
+	<p>{{ description }}</p>
+</article>
+{% endblock %}
+```
+
+### Updating Home Route
+
+After creating the templates, update the *app.py* file:
+
+```py
+from flask import Flask // [!code --]
+from flask import Flask, render_template // [!code ++]
+import directus // [!code ++]
+
+app = Flask(__name__)
+
+
+@app.get("/")
+def home():
+    return "Hello world" // [!code --]
+    global_data = directus.get_global_data() // [!code ++]
+// [!code ++]
+    return render_template( // [!code ++]
+        "home.html", title=global_data["title"], description=global_data["description"] // [!code ++]
+    ) // [!code ++]
+```
+
+Then go to [http://localhost:3000](http://localhost:3000) and you will see a page like this:
+
+![Home page displaying configured global data](home-page.png)
```

Comment:
````suggestion
Create a `templates/home.html` file that will extend the base template and display additional data:

```jinja
{% extends "base.html" %}

{% block content %}
<article>
	<h1>{% block title %}{{ title }}{% endblock %}</h1>
	<p>{{ description }}</p>
</article>
{% endblock %}
```

### Updating Home Route

Update the `app.py` file:

```py
from flask import Flask // [!code --]
from flask import Flask, render_template // [!code ++]
import directus // [!code ++]

app = Flask(__name__)


@app.get("/")
def home():
    return "Hello world" // [!code --]
    global_data = directus.get_global_data() // [!code ++]
// [!code ++]
    return render_template( // [!code ++]
        "home.html", title=global_data["title"], description=global_data["description"] // [!code ++]
    ) // [!code ++]
```

Then go to `http://localhost:3000` in your browser and you will see a page like this:

![Home page displaying configured global data](home-page.png)
````

Lovely lovely lovely. And nice work using the line highlighting - most authors don't bother! 

Most writers start sentences with fillers like 'now', 'next', and 'so'. I tend to remove them because they delay getting to the instruction.